src/main/java/com/gtnewhorizons/angelica/rendering/AngelicaBlockSafetyRegistry.java

```diff
@@ -11,7 +11,7 @@ public class AngelicaBlockSafetyRegistry {
     private static final StampedLock LOCK = new StampedLock();
 
     public static boolean canBlockRenderOffThread(Block block) {
-        long stamp = LOCK.readLock();
+        final long stamp = LOCK.readLock();
```

Comment:
FYI - I don't think it makes a difference to runtime performance/behavior to have `final` here or not.

Reply:
It probably doesn't but I have my ide set to complain about non final and do it as I touch things